vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/samples/registration/batch/SampleRegistrationSpreadsheet.java

```diff
@@ -358,11 +362,18 @@ public enum SamplesheetHeaderName {
         "Sample label"), BIOLOGICAL_REPLICATE_ID("Biological replicate id"), CONDITION(
         "Condition"), SPECIES("Species"), SPECIMEN("Specimen"), ANALYTE(
         "Analyte"), CUSTOMER_COMMENT(
-        "Customer comment");
+        "Customer comment", false);
     public final String label;
+    public final boolean isMandatory;
 
     SamplesheetHeaderName(String label) {
       this.label = label;
+      this.isMandatory = true;
```

Comment:
NIT: I would suggest being explicit with constructors in enums.
You could add `true` everywhere and it would not cost much but when you implicitly assume `true`, the cost of changing it can be high.